### ISSUE TITLE
Reduced contrast in night mode when using a physical keyboard

### ISSUE THREAD

**Issue by a20dev** [chrisboyle/sgtpuzzles#481]
To see the effect:
- enable night mode
- select a puzzle by touching
- the background is black
- hit a key on a physical keyboard (to operate the puzzle)
- the background becomes grey, reducing the contrast

How to avoid the effect 
- enable night mode
- select a puzzle with the physical keyboard

However, the background becomes grey again as soon as the touch screen is touched again (in any app) and the physical keyboard is used  in sgtpuzzles.

The grey background looks like the background of the selected puzzle in the list of puzzles.

In short, any touch event followed by any keyboard event seems to cause the puzzle control become selected and change its background colour from black to grey.

**Comment by a20dev** [comment]
This also happens with the virtual on-screen keyboard.
